src/Disks/DiskLocal.cpp

```diff
@@ -55,9 +58,6 @@ static void loadDiskLocalConfig(const String & name,
             throw Exception("Disk path must end with /. Disk " + name, ErrorCodes::UNKNOWN_ELEMENT_IN_CONFIG);
     }
 
-    if (!FS::canRead(path) || !FS::canWrite(path))
```

Comment:
I missed this change... does it prevent attaching tables and doing SELECTs from readonly fs? We don't need it.

Reply:
Do you mean we should skip the check of `FS::canWrite(disk_path)`? I guess we can mark it read only and also set the available space to zero to prevent any write intentions.

Reply:
Yes, that will make sense.